### ISSUE TITLE
Explore projects does not show all projects correctly.

### ISSUE THREAD

**Issue by camelspeed** [Project-Heimdall/home#12]
The Explore view has the following problems:

1. Limits the projects to 20 like the Popular View.  If these views are sharing technical logic then they must be parameterized such that the features are independent of each other and behave correctly.  There are No Limits right now.  Once we have pagination or infinity scrolling in place we'll revisit the behavior but for now it should grab all projects.
2. All 3 Orgs and their repos must be shown.
3. Projects should be sorted alphanumerically by Project Name.  Currently there is no sort thus the results are nondeterministic (e.g. sometimes projects from one Org show other times from another org).
4. We are currently pulling this data directly from github with each call and that will change in another issue once we assess the design properly.

QUESTION: Once we determine which projects need to change we'll create a new issue in each of their respective boards to track.  Please discuss before starting work.
